washingtonpost/FastFEC
# feat: restore vendored pcre option

## Description

Fixes #44. Adds an option to use the vendored pcre that defaults to true. For installation with package managers like Homebrew, vendored_pcre will be set to false to use the included package.

## Test Steps

These commands all work as expected:

* `zig build`
* `zig build -Dvendored-pcre=true`
* `zig build -Dvendored-pcre=false` (ensure you have installed pcre on the system, e.g. `brew install pcre`)
